docs/orchestration/concepts/api.md

```diff
@@ -51,40 +51,51 @@ client.graphql(
 
 ### Authenticating the Client with Cloud <Badge text="Cloud"/>
 
-If using Prefect Cloud, authentication is required. For interactive use, the most common way to use the Cloud Client is to generate an API key and provide it to the client. After doing so, users can save the API key so it persists across all Python sessions:
+If using Prefect Cloud, authentication is required. See the [API key documentation](./api_keys.md) for instructions on creating and using API keys.
+
+We recommend using the CLI to manage authentication, but authentication may be passed directly to the `Client` as well.
+
+The API key can be passed directly to the client:
 
 ```python
 import prefect
-client = prefect.Client(api_token="API_KEY")
-client.save_api_token()
+
+client = prefect.Client(api_key="API_KEY")
 ```
 
-Now, starting a client in another session will automatically reload the token:
+Since API keys can be used across tenants if linked to a user account, you may also pass a tenant:
 
 ```python
-client = prefect.Client()
-assert client._api_token == "API_KEY"  # True
+client = prefect.Client(api_key="API_KEY", tenant_id="<id>")
 ```
 
-Note that an API key can be provided by environment variable (`PREFECT__CLOUD__AUTH_TOKEN`) or in your Prefect config (under `cloud.auth_token`).
+If you do not pass a tenant, it will be left as `None` which means the default tenant associated with the API key will be used for requests. In that case, you can get the default tenant associated with the key from the client:
 
-Once provisioned with an API key, the Cloud Client can query for available tenants and login to those tenants. In order to query for tenants, call:
+```python
+client.get_default_tenant()
+```
+
+The tenant id can be changed on an existing client, but requests will fail if the API key is not valid for that tenant:
 
 ```python
-client.get_available_tenants()
```

Comment:
is there an easy way to get available tenants in the new system? as a user it may be very difficult to find my tenant ids

Reply:
`client.get_available_tenants()` and `prefect auth list-tenants` both still work wonderfully. This was removed from this section as this was previously a _requirement_ to the login process e.g.

- Set auth token
- List tenants
- Login to tenant

Now it's just

- Set API key

- (Optional) Also set a tenant